### DISCUSSION TITLE
MAC Address Fix

### DISCUSSION THREAD

**Discussion by GregoryHlavac** [pgale/lineage_15.1_Installation_TSV#17] [Ideas]
Found over on https://github.com/dinki/View-Assist/wiki/Lenovo-ThinkSmart#change-mac-address-of-wifi-interface-on-the-lenovo-thinksmart-device-credit-mllockwood

Perhaps include a similar memo below/above the homeassistant setup part since if you have more than one it may be useful?

**Comment by pgale** [comment]
Thanks. I'm aware of those instructions which are for A11 (which has other issues). I originally had instructions to change the MAC address but removed them as I found they didn't work (as did others) and also, Lineage sets a unique MAC address on every device I've tried so far. That has been reported by other installers too. So I don't think this is needed (and probably doesn't work anyway). 

Thanks for the comment though.

**Reply by GregoryHlavac** [reply]
Didn't the bottom of the page say that it explicitly had this issue, or is that not correct anymore/has been changed?

**Reply by pgale** [reply]
Good spot - that was from the original notes from Deadman who co-developed the lineage image. It's incorrect. Just removed it. Thanks.
